packages/ui-patterns/src/patterns/login/login.html

```diff
@@ -0,0 +1,29 @@
+<div
```

Comment:
We could simplify this layout and remove some divs/styles by utilizing CSS grids.

```html
<div class="grid grid-rows-6 gap-y-8">
<!-- layout -->
```

Could you try that out? That way we would not need that many divs and also not that many padding/margins.